### PR TITLE
Fieldtype values editing support

### PR DESCRIPTION
**Pull request by bdunogier**
> JIRA: http://jira.ez.no/browse/EZP-24339
### Contents

~~BDD coverage for User (will fail until implementation is merged from #83) and TextLine fieldtypes~~(moved to #83)
### Other pull-requests
- [x] FieldType form BDD context
- [x] Checkbox support (#82)
- [x] Selection support (#84)
- [x] User support (#83)

### CODE REVIEW

File: features/Context/FieldTypeFormContext.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        );
+    }
+
+    /**
+     * @Then /^the edit form should contain a fieldset named after the field definition$/
+     */
+    public function theEditFormShouldContainAFieldsetNamedAfterTheFieldDefinition()
+    {
+        $this->assertElementContainsText(
+            sprintf('div.ezfield-identifier-%s fieldset legend', self::$fieldIdentifier),
```
Comment (yannickroger) [inline features/Context/FieldTypeFormContext.php]: The selector can be used without the `div`

**Comment by yannickroger** [comment]
+1

**Comment by bdunogier** [comment]
No real need to +1 the whole PR yet, as the topic branches aren't merged yet. But I'll take that as a +1 on the BDD :-)

**Comment by bdunogier** [comment]
All todos taken care of, and tests are passing. What do you say we merge this, @glye ?

**Comment by glye** [comment]
I say not nay, I say aye! :tophat:

**Comment by bdunogier** [comment]
And here we merge.

**Pull request merged**